sunkit_pyvista/plotter.py

```diff
@@ -73,6 +73,24 @@ def set_camera_coordinates(self, coord):
         pos = tuple(camera_position[0])
         self.plotter.camera.position = pos
 
+    def set_view_angle(self, angle: u.deg = 30):
+        """
+        Sets the view angle of the camera by the specified value
+        in degrees.
+
+        Parameters
+        ----------
+        angle : `astropy.units.Quantity`
+            The viewing angle.
+        """
+        view_angle = angle.to_value()
+        if not view_angle > 0 and view_angle <= 180:
+            raise ValueError("specified view angle must be "
+                             "0 deg < angle <= 180 deg")
+
+        zoom_value = 30 / view_angle
```

Comment:
What is the context for `30 /`?

Reply:
Default view angle is set to 30 deg.
But view angle can be altered by set zoom property. Where default zoom is 1 which corresponds to 30 deg, 2 is 15deg and so on. Basically zoom is 30/view_angle

Reply:
Might be worth a comment here explaining this for future reference.